### PR TITLE
Rishi microsoft

### PR DESCRIPTION
**Pull request by Rishikesh-microsoft**
* Removed the variable name timeRequired.

### CODE REVIEW

**Review by chengch801214 — APPROVED** [review]

**Comment by chengch801214** [comment]
I think you need to increase the package.json version?

**Comment by chengch801214** [comment]
I think you need to increase the package.json version?

**Review by chengch801214 — CHANGES_REQUESTED** [review]
I think you need to change the package.json version?

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "name": "tsiclient",
     "main": "tsiclient.js",
     "description": "",
-    "version": "1.4.13",
+    "version": "1.4.14",
```
Comment (chengch801214) [inline package.json:5]: Can you change it to 1.4.17? The latest in the npm registry is 1.4.16.

**Review by chengch801214 — APPROVED** [review]

**Review by rohitams — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
     "name": "tsiclient",
     "main": "tsiclient.js",
     "description": "",
-    "version": "1.4.13",
+    "version": "1.4.17",
```
Comment (rohitams) [inline package.json:5]: Why is this bumped by 4?

Comment (thomasricci) [reply package.json:5]: seems 1.4.16 is the latest published version here: https://www.npmjs.com/package/tsiclient

**Review by thomasricci — APPROVED** [review]

**Pull request merged**